emily/lambda/src/config.rs

```diff
@@ -0,0 +1,36 @@
+use aws_sdk_dynamodb::Client;
+
+/// Temporary chainstate Table name.
+pub static CHAINSTATE_TABLE_NAME: &str = "ChainstateTable-xxxxxxxxxxxx-us-west-2-local";
+
+/// Temporary withdrawal Table name.
+pub static WITHDRAWAL_TABLE_NAME: &str = "WithdrawalTable-xxxxxxxxxxxx-us-west-2-local";
+
+/// Temporary deposit Table name.
+pub static DEPOSIT_TABLE_NAME: &str = "DepositTable-xxxxxxxxxxxx-us-west-2-local";
+
+/// Emily lambda settings.
+#[derive(Debug)]
+pub struct Settings {
+    /// Whether the Emily lambda is running locally.
+    pub is_local: bool,
```

Comment:
This is slightly more complicated than you think it'll be because of some weirdness in the way AWS Lambda spins up locally, so I'm going to put this as a TODO.